includes/wcs-resubscribe-functions.php

```diff
@@ -26,9 +26,10 @@ function wcs_order_contains_resubscribe( $order ) {
 		$order = wc_get_order( $order );
 	}
 
-	$related_subscriptions = wcs_get_subscriptions_for_resubscribe_order( $order );
+	// Pluck the `_subscription_resubscribe` meta key from the order
+	$related_subscription_ids = WCS_Related_Order_Store::instance()->get_related_subscription_ids( $order, 'resubscribe' );
 
-	if ( wcs_is_order( $order ) && ! empty( $related_subscriptions ) ) {
```

Comment:
Same feedback here regarding the `wcs_is_order()` check.

Reply:
Fixed in https://github.com/Automattic/woocommerce-subscriptions-core/commit/ad40418779131969ca359859e2dce84592e317f8